### PR TITLE
US12756: Pagination

### PR DESCRIPTION
**Pull request by lukemoderwell**
_PR does:_
Pagination Stuff
- enables pagination 🎉 
- sets a max 16 posts to each page
- basic pagination styling
- removes the featured post styling (due to conflicts with pagination)

Misc fixes 
- links author names on topic pages
- creates absolutely positioned card blocks (per Adam George's feedback)

### CODE REVIEW

File: collections/_topics/money.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: topic
+name: money
+permalink: /topics/money/
+image: https://images.unsplash.com/photo-1500496733680-167c3db69389?ixlib=rb-0.3.5&s=0bb120921b4af2e0dcd01aa082aee04a&auto=format&fit=crop&w=1239&q=80
```
Comment (tcmacdonald) [inline collections/_topics/money.md]: @lukemoderwell This is going to be tough to populate– we might need to come up with another solution for topic images.

File: _includes/_pagination.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{% if paginator.total_pages > 1 %}
+<div class="pagination text-center" style="width: 100%;">
+  {% if paginator.previous_page %}
+    <a href="{{ paginator.previous_page_path }}" class="previous">Previous</a>
+  {% else %}
+    <span style="color:#979797">Previous</span>
```
Comment (tcmacdonald) [inline _includes/_pagination.html]: Inline styles?

File: collections/_topics/money.md (hunk 1 above)
Comment (lukemoderwell) [reply collections/_topics/money.md]: I'll scrap these for now until we come up with a good way to get images.

Comment (seancdavis) [reply collections/_topics/money.md]: Looks like this is still hanging around.

File: _scss/_base.scss [hunk 3]
```diff
@@ -2,6 +2,13 @@
   flex-wrap: wrap;
 }
 
+.cd_subheading {
```
Comment (seancdavis) [inline _scss/_base.scss]: Is there a reason you went with an underscore instead of hyphen?

File: _config.yml [hunk 4]
```diff
@@ -12,20 +19,20 @@ plugins:
   - jekyll-feed
   - jekyll-redirect-from
   - jekyll-hubspot
+  - jekyll-paginate-v2
+
+pagination:
+  enabled: true
+  debug: false
+  collection: 'posts'
+  per_page: 8
```
Comment (seancdavis) [inline _config.yml]: I'm guessing this is just for demonstration purposes since we don't have 16 posts yet?

File: _scss/_base.scss (hunk 3 above)
Comment (lukemoderwell) [reply _scss/_base.scss]: bad old `BEM` habit. Good catch

File: _config.yml (hunk 4 above)
Comment (lukemoderwell) [reply _config.yml]: Right. It was more for testing.

File: collections/_topics/money.md (hunk 1 above)
Comment (lukemoderwell) [reply collections/_topics/money.md]: I left that and changed the front end to show a default image for all topics until we get that sorted out.

File: _scss/_base.scss (hunk 3 above)
Comment (tcmacdonald) [reply _scss/_base.scss]: We have some loose direction [here](https://design.crossroads.net/developers/getting-started/coding-style) but nothing super specific. I'm guessing @seancdavis is speaking to general convention in `crds-styles`.

**Pull request merged**